docs/source/data_extraction.rst

```diff
@@ -81,3 +82,58 @@ For example:
          preview_roots = [
                LinkByUID(scope="products", id="best cookie ever"),
                LinkByUID(scope="products", id="worst cookie ever")])
+
+Available Row Definitions
+------------------------------
+
+Currently, Ara provides only a single way to define Rows: by the :class:`~taurus.entity.templates.MaterialTemplate` of the roots of the material histories that correspond to each row.
+
+:class:`~citrine.ara.rows.MaterialRunByTemplate`
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The :class:`~citrine.ara.rows.MaterialRunByTemplate` class defines Rows through a list of :class:`~taurus.entity.templates.MaterialTemplate`.
+Every :class:`~taurus.entity.object.MaterialRun` that is assigned to any template in the list is used as the root of a Row.
```

Comment:
```suggestion
Every :class:`~taurus.entity.object.MaterialRun` that is assigned to any template in the list is used as the root of a  Material History to be mapped to a Row.
```

Reply:
Writing about this makes me realize that `Row` is a bit of an awkward name because it conjures an impression of a singular entity in a table. What about renaming to `RowMapper` or something to that effect?

Reply:
How would you define a `RowMapper`?

Reply:
"A RowMapper object describes a mapping from a list of datasets to rows of a table by selecting a set of Material Histories."

Reply:
As an alternative to `RowDefinition`?